rootfs/scheduler/__init__.py

```diff
@@ -1049,6 +1051,53 @@ def _healthcheck(self, controller, routable=False, path='/', port=5000, delay=30
 
         return controller
 
+    '''
+    Applies exec readiness probe to the slugrunner container.
+    http://kubernetes.io/docs/user-guide/pod-states/#container-probes
+
+    /runner/init is the entry point of the slugrunner.
+    https://github.com/deis/slugrunner/blob/master/rootfs/Dockerfile#L24
```

Comment:
It might be better to leave such specific GitHub links out of the actual commit. They work now, but may point to entirely different lines of code soon.